composer.json

```diff
@@ -20,34 +20,34 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.4",
         "sensiolabs/security-checker": " ^5.0",
-        "sylius/sylius": "^1.5",
-        "symfony/flex": "^1.2"
+        "friendsofsymfony/oauth-server-bundle": ">2.0.0-alpha.0 ^2.0@dev",
+        "sylius/sylius": "^1.9",
+        "symfony/flex": "^1.7"
     },
     "require-dev": {
-        "behat/behat": "^3.4",
-        "behat/mink": "^1.7@dev",
-        "behat/mink-browserkit-driver": "^1.3",
-        "behat/mink-extension": "^2.2",
+        "behat/behat": "^3.6.1",
         "behat/mink-selenium2-driver": "^1.3",
+        "friends-of-behat/mink": "^1.8",
+        "friends-of-behat/mink-browserkit-driver": "^1.4",
+        "friends-of-behat/mink-extension": "^2.4",
+        "friends-of-behat/mink-debug-extension": "^2.0",
         "friends-of-behat/exclude-specifications-extension": "^0.2",
         "friends-of-behat/page-object-extension": "^0.3",
-        "friends-of-behat/suite-settings-extension": "^1.0",
-        "friends-of-behat/symfony-extension": ">=2.0 <=2.0.6",
-        "friends-of-behat/variadic-extension": "^1.1",
-        "lakion/mink-debug-extension": "^1.2.3",
-        "lchrusciel/api-test-case": "^3.0",
-        "phpspec/phpspec": "^5.0",
-        "phpunit/phpunit": "^6.5",
+        "friends-of-behat/symfony-extension": "^2.1",
+        "friends-of-behat/variadic-extension": "^1.3",
+        "lchrusciel/api-test-case": "^5.0",
+        "phpspec/phpspec": "^7.0",
+        "phpunit/phpunit": "^8.5",
         "stripe/stripe-php": "^4.1",
         "sylius-labs/coding-standard": "^2.0",
-        "symfony/browser-kit": "^3.4|^4.1",
-        "symfony/debug-bundle": "^3.4|^4.1",
-        "symfony/dotenv": "^3.4|^4.1",
-        "symfony/intl": "^3.4|^4.1",
-        "symfony/web-profiler-bundle": "^3.4|^4.1",
-        "symfony/web-server-bundle": "^3.4|^4.1"
+        "symfony/browser-kit": "^4.4",
```

Comment:
Is it a big problem to update to 5.2?

Reply:
`symfony/browser-kit`,  `symfony/dotenv`, `symfony/intl` is possible to update to 5.2, but `symfony/debug-bundle`, `symfony/web-profiler-bundle`, `symfony/web-server-bundle` have conflict with few test libs 

<img width="1440" alt="Screenshot 2021-03-15 at 11 04 37" src="https://user-images.githubusercontent.com/39232096/111136673-467e7c00-857e-11eb-9082-e5ee1e023123.png">


after update symfony-depedency-injection to higher version, I have other problems with for example: `friends-of-behat/exclude-specifications-extension`